app/src/main/java/com/mapbox/mapboxsdk/plugins/testapp/activity/location/LocationLayerModesActivity.java

```diff
@@ -93,6 +96,7 @@ public void onMapReady(MapboxMap mapboxMap) {
     locationEngine.activate();
     locationLayerPlugin = new LocationLayerPlugin(mapView, mapboxMap, locationEngine);
     locationLayerPlugin.setOnLocationClickListener(this);
+    locationLayerPlugin.setLocationLayerEnabled(true);
```

Comment:
can this be the default behaviour, so no additional setter need to be called after using the constructor?